src/main/java/fi/metatavu/dcfb/server/rest/ItemsApiImpl.java

```diff
@@ -117,35 +126,43 @@ public Response findItem(UUID itemId) throws Exception {
   }
 
   @Override
-  public Response listItems(String categoryIdsParam, String search, List<String> sort, Long firstResult, Long maxResults) throws Exception {
+  public Response listItems(String categoryIdsParam, String locationIdsParam, String search, List<String> sort, Long firstResult, Long maxResults) throws Exception {
     List<Category> categories = null;
+    List<Location> locations = null;
 
-    if (categoryIdsParam != null) {
-      try {      
-        categories = Arrays.stream(StringUtils.split(categoryIdsParam, ','))
-          .map(UUID::fromString)
-          .map(categoryId -> {
-            Category category = categoryController.findCategory(categoryId);
-            if (category == null) {
-              throw new IllegalArgumentException(String.format("Could not find category %s", categoryId));
-            }
-            
-            return category;
-          })
-          .collect(Collectors.toList());
-      } catch (IllegalArgumentException e) {
-        return createBadRequest(e.getMessage());
-      }
-    }
+    try {
+      categories = getListParameter(categoryIdsParam, (param) -> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the parentheses around the "param" parameter ([squid:S1611](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1611))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=11&issues=AWQt_bhgYwZU9vvrs3Kv&open=AWQt_bhgYwZU9vvrs3Kv)

[ISSUE_KEY]: AWQt_bhgYwZU9vvrs3Kv